packages/smart-contracts/src/lib/artifacts/RequestDeployer/index.ts

```diff
@@ -0,0 +1,64 @@
+import { ContractArtifact } from '../../ContractArtifact';
+
+import { abi as ABI_0_1_0 } from './0.1.0.json';
+// @ts-ignore Cannot find module
+import type { RequestDeployer } from '../../../types/RequestDeployer';
+
+export const requestDeployer = new ContractArtifact<RequestDeployer>(
+  {
+    '0.1.0': {
```

Comment:
Probably input for later, but important:

The deployer is the contract that should be deployed to the same address on every network (except for private), and it does not need a `creationBlockNumber` because we don't need to listen to events. So it could be the opportunity to initiate a new artifact template with something like

```
deployment: {
private: {
          address: '0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0',
          creationBlockNumber: 0,
},
_live: {
          address: '0xE99Ab70a5FAE59551544FA326fA048f7B95A24B2',
}
```